File: unittest/scripts/auto/py_shell/scripts/shell_help_norecord.py
#@ shell
shell.help()

#@ global ? for Shell[USE:shell]
\? Shell

#@ global help for Shell[USE:shell]
\help Shell

#@ shell.add_extension_object_member
shell.help('add_extension_object_member')

#@ global ? for add_extension_object_member[USE:shell.add_extension_object_member]
\? Shell.add_extension_object_member

#@ global help for add_extension_object_member[USE:shell.add_extension_object_member]
\help Shell.add_extension_object_member

#@ shell.connect
shell.help('connect')

#@ global ? for connect[USE:shell.connect]
\? Shell.connect

#@ global help for connect[USE:shell.connect]
\help Shell.connect

#@ shell.create_extension_object
shell.help('create_extension_object')

#@ global ? for create_extension_object[USE:shell.create_extension_object]
\? Shell.create_extension_object

#@ global help for create_extension_object[USE:shell.create_extension_object]
\help Shell.create_extension_object

#@ shell.delete_all_credentials
shell.help('delete_all_credentials')

#@ global ? for delete_all_credentials [USE:shell.delete_all_credentials]
\? Shell.delete_all_credentials

#@ global help for delete_all_credentials [USE:shell.delete_all_credentials]
\help Shell.delete_all_credentials

#@ shell.delete_credential
shell.help('delete_credential')

#@ global ? for delete_credential [USE:shell.delete_credential]
\? Shell.delete_credential

#@ global help for delete_credential [USE:shell.delete_credential]
\help Shell.delete_credential

#@ shell.disable_pager
shell.help('disable_pager')

#@ global ? for disable_pager [USE:shell.disable_pager]
\? Shell.disable_pager

#@ global help for disable_pager [USE:shell.disable_pager]
\help Shell.disable_pager

#@ shell.enable_pager
shell.help('enable_pager')

#@ global ? for enable_pager [USE:shell.enable_pager]
\? Shell.enable_pager

#@ global help for enable_pager [USE:shell.enable_pager]
\help Shell.enable_pager

#@ shell.get_session
shell.help('get_session')

#@ global ? for get_session[USE:shell.get_session]
\? Shell.get_session

#@ global help for get_session[USE:shell.get_session]
\help Shell.get_session

#@ shell.help
shell.help('help')

#@ global ? for help[USE:shell.help]
\? Shell.help

#@ global help for help[USE:shell.help]
\help Shell.help

#@ shell.list_credential_helpers
shell.help('list_credential_helpers')

#@ global ? for list_credential_helpers [USE:shell.list_credential_helpers]
\? Shell.list_credential_helpers

#@ global help for list_credential_helpers [USE:shell.list_credential_helpers]
\help Shell.list_credential_helpers

#@ shell.list_credentials
shell.help('list_credentials')

#@ global ? for list_credentials [USE:shell.list_credentials]
\? Shell.list_credentials

#@ global help for list_credentials [USE:shell.list_credentials]
\help Shell.list_credentials

#@ shell.log
shell.help('log')

#@ global ? for log[USE:shell.log]
\? Shell.log

#@ global help for log[USE:shell.log]
\help Shell.log

#@ shell.options
shell.help('options')

#@ global ? for options[USE:shell.options]
\? Shell.options

#@ global help for options[USE:shell.options]
\help Shell.options

#@ shell.parse_uri
shell.help('parse_uri')

#@ global ? for parse_uri[USE:shell.parse_uri]
\? Shell.parse_uri

#@ global help for parse_uri[USE:shell.parse_uri]
\help Shell.parse_uri

#@ shell.prompt
shell.help('prompt')

#@ global ? for prompt[USE:shell.prompt]
\? Shell.prompt

#@ global help for prompt[USE:shell.prompt]
\help Shell.prompt

#@ shell.reconnect
shell.help('reconnect')

#@ global ? for reconnect[USE:shell.reconnect]
\? Shell.reconnect

#@ global help for reconnect[USE:shell.reconnect]
\help Shell.reconnect

#@ shell.reports
shell.help('reports')

#@ global ? for reports [USE:shell.reports]
\? Shell.reports

#@ global help for reports [USE:shell.reports]
\help Shell.reports

#@ shell.register_global
shell.help('register_global')

#@ global ? for register_global[USE:shell.register_global]
\? Shell.register_global

#@ global help for register_global[USE:shell.register_global]
\help Shell.register_global

#@ shell.register_report
shell.help('register_report')

#@ global ? for register_report [USE:shell.register_report]
\? Shell.register_report

#@ global help for register_report [USE:shell.register_report]
\help Shell.register_report

#@ shell.set_current_schema
shell.help('set_current_schema')

#@ global ? for set_current_schema[USE:shell.set_current_schema]
\? Shell.set_current_schema

#@ global help for set_current_schema[USE:shell.set_current_schema]
\help Shell.set_current_schema

#@ shell.set_session
shell.help('set_session')

#@ global ? for set_session[USE:shell.set_session]
\? Shell.set_session

#@ global help for set_session[USE:shell.set_session]
\help Shell.set_session

#@ shell.status
shell.help('status')

#@ global ? for status[USE:shell.status]
\? Shell.status

#@ global help for status[USE:shell.status]
\help Shell.status

#@ shell.store_credential
shell.help('store_credential')

#@ global ? for store_credential [USE:shell.store_credential]
\? Shell.store_credential

#@ global help for store_credential [USE:shell.store_credential]
\help Shell.store_credential

#@ BUG28393119 UNABLE TO GET HELP ON CONNECTION DATA, before session
\? connection

#@ BUG28393119 UNABLE TO GET HELP ON CONNECTION DATA, after session
shell.connect(__mysqluripwd)
\? connection
session.close()

#@ Help on plugins
\? plugins

#@ Help on extension objects
\? extension objects

#@ unparse_uri
\? unparse_uri

#@ dump_rows
\? dump_rows

File: unittest/scripts/auto/js_shell/scripts/shell_help_norecord.js
//@ Shell Help
shell.help()

//@ Shell Help, \? [USE:Shell Help]
\? shell

//@ Help on Options
shell.help("options")

//@ Help on Options, \? [USE:Help on Options]
\? shell.options

//@ Help on Reports
shell.help("reports")

//@ Help on Reports, \? [USE:Help on Reports]
\? reports

//@ Help on addExtensionObjectMember
shell.help("addExtensionObjectMember")

//@ Help on addExtensionObjectMember, \? [USE:Help on addExtensionObjectMember]
\? addExtensionObjectMember

//@ Help on shell.addExtensionObjectMember, \? [USE:Help on addExtensionObjectMember]
\? shell.addExtensionObjectMember

//@ Help on autoCompleteSql
// WL13397-TSFR_2_1
shell.help("autoCompleteSql")

//@ Help on autoCompleteSql, \? [USE:Help on autoCompleteSql]
\? autoCompleteSql

//@ Help on shell.autoCompleteSql, \? [USE:Help on autoCompleteSql]
\? shell.autoCompleteSql

//@ Help on Connect
shell.help("connect")

//@ Help on Connect, \? [USE:Help on Connect]
\? connect

// WL13236-TSFR4_1: Validate that the function named shell.connectToPrimary([connectionData, password]) exists under the shell global object and receives two optional parameters: connectionData and password.
// WL13236-TSFR4_2: Validate that the help system has an entry for the function shell.connectToPrimary([connectionData, password]). Test with: \h shell.connectToPrimary, \h connectToPrimary, \h shell.help('connectToPrimary'), \h shell.

//@ Help on connectToPrimary
shell.help("connectToPrimary")

//@ Help on connectToPrimary, \? [USE: Help on connectToPrimary]
\? connectToPrimary

//@ Help on shell.connectToPrimary [USE: Help on connectToPrimary]
\? shell.connectToPrimary

//@ Help on createExtensionObject
shell.help("createExtensionObject")

//@ Help on createExtensionObject, \? [USE:Help on createExtensionObject]
\? createExtensionObject

//@ Help on shell.createExtensionObject, \? [USE:Help on createExtensionObject]
\? shell.createExtensionObject

//@ Help on createResult
shell.help("createResult")

//@ Help on createResult, \? [USE:Help on createResult]
\? createResult

//@ Help on shell.createResult, \? [USE:Help on createResult]
\? shell.createResult

//@ Help on deleteAllCredentials
shell.help("deleteAllCredentials")

//@ Help on deleteAllCredentials, \? [USE:Help on deleteAllCredentials]
\? deleteAllCredentials

//@ Help on deleteAllSecrets
shell.help("deleteAllSecrets")

//@ Help on deleteAllSecrets, \? [USE: Help on deleteAllSecrets]
\? deleteAllSecrets

//@ Help on deleteCredential
shell.help("deleteCredential")

//@ Help on deleteCredential, \? [USE:Help on deleteCredential]
\? shell.deleteCredential

//@ Help on deleteSecret
shell.help("deleteSecret")

//@ Help on deleteSecret, \? [USE: Help on deleteSecret]
\? shell.deleteSecret

//@ Help on disablePager
shell.help("disablePager")

//@ Help on disablePager, \? [USE:Help on disablePager]
\? disablePager

//@ Help on enablePager
shell.help("enablePager")

//@ Help on enablePager, \? [USE:Help on enablePager]
\? enablePager

//@ Help on getSession
shell.help("getSession")

//@ Help on getSession, \? [USE:Help on getSession]
\? shell.getSession

//@ Help on Help
shell.help("help")

//@ Help on Help, \? [USE:Help on Help]
\? shell.help

//@ Help on listCredentialHelpers
shell.help("listCredentialHelpers")

//@ Help on listCredentialHelpers, \? [USE:Help on listCredentialHelpers]
\? listCredentialHelpers

//@ Help on listCredentials
shell.help("listCredentials")

//@ Help on listCredentials, \? [USE:Help on listCredentials]
\? shell.listCredentials

//@ Help on listSecrets
shell.help("listSecrets")

//@ Help on listSecrets, \? [USE: Help on listSecrets]
\? shell.listSecrets

//@ Help on listSshConnections
shell.help("listSshConnections")

//@ Help on listSshConnections, \? [USE:Help on listSshConnections]
\? listSshConnections

//@ Help on Log
shell.help("log")

//@ Help on Log, \? [USE:Help on Log]
\? log

//@ Help on parseUri
shell.help("parseUri")

//@ Help on parseUri, \? [USE:Help on parseUri]
\? parseuri

//@ Help on Prompt
shell.help("prompt")

//@ Help on Prompt, \? [USE:Help on Prompt]
\? prompt

//@ Help on shell.disconnect
shell.help("disconnect")

//@ Help on disconnect [USE: Help on shell.disconnect]
\? shell.disconnect

//@ Help on readSecret
shell.help("readSecret")

//@ Help on readSecret, \? [USE: Help on readSecret]
\? readSecret

//@ Help on reconnect
shell.help("reconnect")

//@ Help on reconnect, \? [USE:Help on reconnect]
\? reconnect

//@ Help on registerGlobal
shell.help("registerGlobal")

//@ Help on registerGlobal, \? [USE:Help on registerGlobal]
\? registerGlobal

//@ Help on shell.registerGlobal, \? [USE:Help on registerGlobal]
\? shell.registerGlobal


//@ Help on registerReport
shell.help("registerReport")

//@ Help on registerReport, \? [USE:Help on registerReport]
\? registerReport


//@ Help on registerSQLHandler
shell.help("registerSQLHandler")

//@ Help on registerSQLHandler, \? [USE:Help on registerSQLHandler]
\? registerSQLHandler

//@ Help on listSQLHandlers
shell.help("listSQLHandlers")

//@ Help on listSQLHandlers, \? [USE:Help on listSQLHandlers]
\? listSQLHandlers

//@ Help on setCurrentSchema
shell.help("setCurrentSchema")

//@ Help on setCurrentSchema, \? [USE:Help on setCurrentSchema]
\? shell.setCurrentSchema

//@ Help on setSession
shell.help("setSession")

//@ Help on setSession, \? [USE:Help on setSession]
\? setSession

//@ Help on status
shell.help("status")

//@ Help on status, \? [USE:Help on status]
\? shell.status

//@ Help on print
shell.help("print")

//@ Help on print, \? [USE:Help on print]
\? shell.print

//@ Help on storeCredential
shell.help("storeCredential")

//@ Help on storeCredential, \? [USE:Help on storeCredential]
\? storeCredential

//@ Help on storeSecret
shell.help("storeSecret")

//@ Help on storeSecret, \? [USE: Help on storeSecret]
\? storeSecret

//@ BUG28393119 UNABLE TO GET HELP ON CONNECTION DATA, before session
\? connection

//@ BUG28393119 UNABLE TO GET HELP ON CONNECTION DATA, after session
shell.connect(__mysqluripwd)
\? connection
session.close()

//@ Help on connection attributes
\? connection attributes

//@ Help on connection compression
\? connection compression

//@ Help on connection options
\? connection options

//@ Help on connection types
\? connection types

//@ Help on extension objects
\? extension objects

//@ Help on unparseUri
\? unparseUri

//@ Help on dumpRows
\? dumpRows
